apps/script/generateLevelImages.js

```diff
@@ -10,7 +10,7 @@ if (args.indexOf("--help") !== -1 || args.indexOf("-h") !== -1) {
   console.log("Options:");
   console.log("\t-h, --help\tprint this usage message");
   console.log("\t--grid-only\tgenerate only maze grid images for only those levels that have them");
-  phantom.exit();
+  phantom.exit(); // eslint-disable-line no-undef
```

Comment:
Recommend instead of these one-offs, put the following at the top of the file:

```
/* eslint-env node, phantomjs */
```

See [specifying environments](http://eslint.org/docs/user-guide/configuring#specifying-environments).